drivers/hmis/app/graphql/mutations/refresh_external_submissions.rb

```diff
@@ -0,0 +1,20 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
+  class RefreshExternalSubmissions < CleanBaseMutation
+    field :success, Boolean, null: false
+
+    def resolve
+      handlers = ['HmisExternalApis::ConsumeExternalFormSubmissionsJob']
+      return if Delayed::Job.queued?(handlers) || Delayed::Job.running?(handlers)
+
+      queue = ENV.fetch('DJ_SHORT_QUEUE_NAME', :short_running)
+      HmisExternalApis::ConsumeExternalFormSubmissionsJob.delay(priority: 12, queue: queue)
```

Comment:
What should priority be here? I just copied this from the service history queuer but maybe this should be higher priority?

Reply:
Ah hm maybe priority should be nil/not passed, if priority is an alternative to queue as described here: https://github.com/collectiveidea/delayed_job?tab=readme-ov-file#parameters

Reply:
I think it should be high priority (`priority: 0`) since it was a user-initiated action

Reply:
It looks like we also have negative numbers for even-higher priority https://github.com/greenriver/hmis-warehouse/blob/f039ec719577065e8d3c7c0987b0ee3e3a131c40/config/initializers/delayed_job.rb#L9-L11